### PR TITLE
graphql: Use relative imports and mark unused file

### PR DESCRIPTION
**Pull request by ghost**
Should fix the tester (along with Microsoft/types-publisher#281). Broken in  #13831.
@TonyPythoneer The `UNUSED_FILES.txt` is a TODO. It's because the tester noticed that this file isn't imported anywhere.

### CODE REVIEW

**Comment by TonyPythoneer** [comment]
@andy-ms OMG. I am sorry about this. How do I help this?

**Comment by andy-ms** [comment]
It's already merged so things are fixed for now. But `execution/values.d.ts` is currently dead code. Did you mean to re-export that from somewhere?

**Comment by TonyPythoneer** [comment]
Let me know. Does dead code mean it doesn't reuse by other file? If it's right. Let's keep going as the following.

I just found source code. E.g. in getVariableValues, it's used in another function scope as validatation.
https://github.com/graphql/graphql-js/search?utf8=✓&q=getVariableValues
https://github.com/graphql/graphql-js/blob/4a832566c9eb5e1633ef0724b74e86039c148539/src/execution/execute.js#L241

I think the tester request each subfile is re-used or to re-export for somewhere. Otherwise, it will be consindered as independent or main file (ideally, index.d.ts should the main file). Therefore, it's maybe a hidden fault reason for tester.

My opinion is to keep the type according to source code. Everything keeps like origin. But I found it's not to export for users to use, the solution is to delete the file and avoid user to use that.

Actually, I am not familiar with graphql. It's because the offical document is not clear. It needs to reslove source code and know what code can be used by user.

To sum up, we can dicuss how to meet the source code and this community.

**Comment by andy-ms** [comment]
If a function isn't part of the public API, there shouldn't be typings for it. It may just be an internal implementation detail.

**Comment by TonyPythoneer** [comment]
Sure, if there is no public API in the file, I think it should be deleted.

Yes, graphql has been typed by flow. But I don't know how to access those types because I had choice typescript as type system, I haven't cared flow.

Currently, this type file depends on manual translation, it's from flow to typescript, the achievement is made by contributors. Suppose typescript is able to access flow type and output `.d.ts` file, it's ideal and easy method, but it will bring extra work for typescript team. If you have any good idea, it can improve this, please tell me. Thanks.
